### PR TITLE
Add the dcrwallet tagged version v3

### PR DESCRIPTION
**Pull request by dmigwi**
Closes https://github.com/crypto-power/cryptopower/issues/7

Updated the dcrwallet to release https://github.com/decred/dcrwallet/releases/tag/release-v1.8.0 using the version tagged at [v3.0.1](https://github.com/decred/dcrwallet/releases/tag/v3.0.1)

### CODE REVIEW

**Comment by crux25** [comment]
You only upgrade the dcrwallet module, how about other modules?

**Comment by dmigwi** [comment]
> How about upgrading the dcrd modules to https://github.com/decred/dcrd/releases/tag/release-v1.8.0 on this PR.

We don't utilize dcrd directly on cryptopower, updating the dcrwallet to the latest release does use latest dcrd release.

**Comment by crux25** [comment]
> > How about upgrading the dcrd modules to https://github.com/decred/dcrd/releases/tag/release-v1.8.0 on this PR.
> 
> We don't utilize dcrd directly on cryptopower, updating the dcrwallet to the latest release does use latest dcrd release.

Looked at the go.mod and those are updated as well.

**Review by crux25 — APPROVED** [review]

**Review by dreacot — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -70,18 +54,22 @@ require (
...
-	decred.org/cspp/v2 v2.0.0 // indirect
-	decred.org/dcrwallet v1.7.1
-	github.com/companyzero/sntrup4591761 v0.0.0-20200131011700-2b0d299dbd22 // indirect
-)
-
-require github.com/crypto-power/instantswap v0.0.0-20230619161136-95ecf47d5ebf
-
-require (
+	decred.org/cspp/v2 v2.1.0 // indirect
+	decred.org/dcrwallet v1.7.0 // indirect
```
Comment (dreacot) [inline go.mod:71]: @dmigwi why is this on 1.7.0

Comment (dmigwi) [reply go.mod:71]: That is automatically imported by the dependencies. Let me look at which dependency is causing this indirect use.

Comment (dmigwi) [reply go.mod:71]: It turns out that dcrdata hasn't been updated we use it a dependency. https://github.com/decred/dcrdata/tags

 I will try and phase it out.

**Pull request merged**